src/Apps/ArtQuiz/ArtQuizResultsLoader.tsx

```diff
@@ -0,0 +1,70 @@
+import {
+  Box,
+  Flex,
+  Text,
+  Spacer,
+  ArtsyMarkBlackIcon,
+  FullBleed,
+  Spinner,
+} from "@artsy/palette"
+import { SplitLayout } from "Components/SplitLayout"
+import { useNavBarHeight } from "Components/NavBar/useNavBarHeight"
+import { useState, useEffect } from "react"
+import { useTranslation } from "react-i18next"
+
+export const ArtQuizResultsLoader = () => {
+  const { desktop } = useNavBarHeight()
+  const { t } = useTranslation()
+
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
+    setTimeout(() => {
+      setLoading(false)
+    }, 2000)
+  }, [])
+
+  return (
+    <FullBleed height={`calc(100vh - ${desktop}px)`}>
+      <Box height="100%" padding={0}>
+        <SplitLayout
+          hideLogo
+          left={
+            <Flex height="100%" alignItems="center" justifyContent="center">
+              <ArtsyMarkBlackIcon height="65px" width="65px" fill="white100" />
+            </Flex>
+          }
+          leftProps={{ display: ["none", "block"] }}
+          right={
+            <Flex
+              width="100%"
+              flexDirection="column"
+              justifyContent="center"
+              alignItems="center"
+              p={[2, 4]}
+            >
+              {/* Vertically centers next Box */}
+
+              {loading ? (
+                <Spinner position="inherit" display="flex" />
+              ) : (
+                <Flex height="6px" />
+              )}
+              <Spacer my={2} />
+              <Text variant={["lg", "xl"]}> {t("artQuizPage.title")}</Text>
+
+              <Spacer my={2} />
+
+              <Text variant={["sm", "md"]} color="black60">
+                {loading
+                  ? t("artQuizPage.loadingScreen.calculatingResults")
+                  : t("artQuizPage.loadingScreen.resultsComplete")}
+                <Spacer my={4} />
```

Comment:
We typically try to avoid inner margins if possible. Any way this can be hoisted up? Example:

```
// Avoid 
const App = () => {
  return (
    <>
      <Foo />
      <Bar /> 
    </>
  )
}

const Foo = () => {
  return (
    <>
      <Box>Hello!</>
      <Spacer my={2} />
    </>
  )
}

const Bar = () => <Box>World</Box>
```
Better:
```tsx
const App = () => {
  return (
    <>
      <Foo />
       <Spacer my={2} /> 
      <Bar /> 
    </>
  )
}

const Foo = () => {
  return (
    <>
      <Box>Hello!</>
    </>
  )
}
...
```

Or even:
```tsx
```tsx
const App = () => {
  return (
    <>
      <Foo my={2} />
      <Bar /> 
    </>
  )
}
```

The reasoning here is that components that don't manage their own margins are portable, and layouts should always be managed from above.

Reply:
(Speaking of which, I should update the FAQ page)

Reply:
resolved with [9bf44d8](https://github.com/artsy/force/pull/11146/commits/9bf44d8e7cdce3115c041f5a1f56fd7fbcb29774)